diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter5/LambdaParameterOrder.kt

```diff
@@ -28,7 +29,16 @@ class LambdaParameterOrder(configRules: List<RulesConfig>) : DiktatRule(
     @Suppress("UnsafeCallOnNullableType")
     private fun checkArguments(node: ASTNode) {
         val funArguments = (node.psi as KtFunction).valueParameters
-        val sortArguments = funArguments.sortedBy { it.typeReference?.node?.hasChildOfType(FUNCTION_TYPE) }
+        val sortArguments = funArguments
+            .sortedBy {
+                it
+                    .typeReference
+                    ?.node
+                    ?.let { astNode ->
+                        astNode.findChildByType(NULLABLE_TYPE) ?: astNode
```

Comment:
Kotlin best practices are to make outer lambda's argument named. It's strange, actually, that we still don't check it in diktat

Reply:
#1009